x-pack/plugins/cloud_security_posture/public/common/utils/get_dataset_display_name.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+type Dataset = 'wiz.cloud_configuration_finding' | 'cloud_security_posture.findings';
+
+export const WIZ_DATASET = 'wiz.cloud_configuration_finding';
+export const CSP_DATASET = 'cloud_security_posture.findings';
+
+export const getDatasetDisplayName = (dataset?: Dataset | string) => {
+  if (dataset === WIZ_DATASET) return 'Wiz';
+  if (dataset === CSP_DATASET) return 'Elastic CSP';
+};
```

Comment:
```suggestion
const datasetMapping: { [key: string]: string } = {
  [WIZ_DATASET]: 'Wiz',
  [CSP_DATASET]: 'Elastic CSP',
};

function getDatasetName(dataset: string): string {
  return datasetMapping[dataset] ;
}

```

Reply:
I've implemented the suggestion, but then pushed a revert. the typing doesnt make sense as much as with a pure function. because we need to support strings which are not mapped, and creating a mapped object which accepts a `string` as key doesnt make much sense.

But thanks for the suggestion regardless